vowpalwabbit/feature_group.h

```diff
@@ -50,179 +57,227 @@ struct features;
 /// iterator over feature values only
 class features_value_iterator
 {
-protected:
+ protected:
   feature_value* _begin;
-public:
-  features_value_iterator(feature_value* begin)
-    : _begin(begin)
-  { }
 
-  features_value_iterator(const features_value_iterator& other)
-    : _begin(other._begin)
-  { }
+ public:
+  features_value_iterator(feature_value* begin) : _begin(begin)
+  {
```

Comment:
I think we should explore if there is an option for this to become 
`{}` instead